### PR TITLE
CLOUDSTACK-9982: Introduce PhysicalSize as response in listTemplates API

### PR DESCRIPTION
**Pull request by HarikaPunna**
Introduce PhysicalSize as response in listTemplates API

### CODE REVIEW

**Comment by kiwiflyer** [comment]
@HarikaPunna Please create a jira for this and add it to the PR subject. Thanks!

**Comment by cloudmonger** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 914
 Hypervisor xenserver
 NetworkType Advanced
 Passed=101
 Failed=10
 Skipped=12

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/r2si930m8xxzavs/AAAzNrnoF1fC3auFrvsKo_8-a?dl=0

**Failed tests:**
* test_scale_vm.py

 * ContextSuite context=TestScaleVm>:setup Failing since 23 runs

* test_deploy_vm_iso.py

 * test_deploy_vm_from_iso Failing since 7 runs

* test_list_ids_parameter.py

 * ContextSuite context=TestListIdsParams>:setup Failing since 32 runs

* test_volumes.py

 * test_06_download_detached_volume Failing since 3 runs

* test_vm_life_cycle.py

 * test_10_attachAndDetach_iso Failing since 7 runs

* test_routers_network_ops.py

 * test_01_isolate_network_FW_PF_default_routes_egress_true Failing since 40 runs

 * test_02_isolate_network_FW_PF_default_routes_egress_false Failing since 40 runs

 * test_01_RVR_Network_FW_PF_SSH_default_routes_egress_true Failing since 40 runs

 * test_02_RVR_Network_FW_PF_SSH_default_routes_egress_false Failing since 40 runs

 * test_03_RVR_Network_check_router_state Failed


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_01_verify_libvirt
test_02_verify_libvirt_after_restart
test_03_verify_libvirt_attach_disk
test_04_verify_guest_lspci
test_05_change_vm_ostype_restart
test_06_verify_guest_lspci_again
test_static_role_account_acls
test_11_ss_nfs_version_on_ssvm
test_nested_virtualization_vmware
test_3d_gpu_support
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_vm_snapshots.py
test_over_provisioning.py
test_global_settings.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_loadbalance.py
test_routers.py
test_reset_vm_on_reboot.py
test_deploy_vms_with_varied_deploymentplanners.py
test_network.py
test_router_dns.py
test_non_contigiousvlan.py
test_login.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_metrics_api.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_disk_offerings.py

File: setup/db/db/schema-4930to41000-cleanup.sql [hunk 1]
```diff
@@ -22,3 +22,105 @@
 DELETE FROM `cloud`.`configuration` WHERE name='consoleproxy.loadscan.interval';
 
 DELETE FROM `cloud`.`host_details` where name = 'vmName' and  value in (select name from `cloud`.`vm_instance`  where state = 'Expunging' and hypervisor_type ='BareMetal');
+
```
Comment (sureshanaparti) [inline setup/db/db/schema-4930to41000-cleanup.sql]: Move the changes to _schema-4930to41000.sql_

**Comment by sureshanaparti** [comment]
Code changes LGTM.

**Comment by sureshanaparti** [comment]
@HarikaPunna Check the failed tests above and re-run BVT tests.

**Comment by borisstoyanov** [comment]
@blueorangutan package

**Comment by borisstoyanov** [comment]
@blueorangutan test

**Comment by cloudmonger** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 924
 Hypervisor xenserver
 NetworkType Advanced
 Passed=104
 Failed=9
 Skipped=12

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/r2si930m8xxzavs/AAAzNrnoF1fC3auFrvsKo_8-a?dl=0

**Failed tests:**
* test_non_contigiousvlan.py

 * test_extendPhysicalNetworkVlan Failed

* test_deploy_vm_iso.py

 * test_deploy_vm_from_iso Failing since 9 runs

* test_volumes.py

 * test_06_download_detached_volume Failing since 5 runs

* test_vm_life_cycle.py

 * test_10_attachAndDetach_iso Failing since 9 runs

* test_routers_network_ops.py

 * test_01_isolate_network_FW_PF_default_routes_egress_true Failing since 42 runs

 * test_02_isolate_network_FW_PF_default_routes_egress_false Failing since 42 runs

 * test_01_RVR_Network_FW_PF_SSH_default_routes_egress_true Failing since 42 runs

 * test_02_RVR_Network_FW_PF_SSH_default_routes_egress_false Failing since 42 runs

 * test_03_RVR_Network_check_router_state Failed


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_01_verify_libvirt
test_02_verify_libvirt_after_restart
test_03_verify_libvirt_attach_disk
test_04_verify_guest_lspci
test_05_change_vm_ostype_restart
test_06_verify_guest_lspci_again
test_static_role_account_acls
test_11_ss_nfs_version_on_ssvm
test_nested_virtualization_vmware
test_3d_gpu_support
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_vm_snapshots.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_loadbalance.py
test_routers.py
test_reset_vm_on_reboot.py
test_deploy_vms_with_varied_deploymentplanners.py
test_network.py
test_router_dns.py
test_login.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_metrics_api.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_disk_offerings.py

**Comment by jayapalu** [comment]
LGTM

**Comment by rhtyd** [comment]
@HarikaPunna LGTM, please move the sql change to 4.10->4.11 upgrade path.

**Comment by cloudmonger** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 977
 Hypervisor xenserver
 NetworkType Advanced
 Passed=103
 Failed=8
 Skipped=12

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/r2si930m8xxzavs/AAAzNrnoF1fC3auFrvsKo_8-a?dl=0

**Failed tests:**
* test_non_contigiousvlan.py

 * test_extendPhysicalNetworkVlan Failed

* test_deploy_vm_iso.py

 * test_deploy_vm_from_iso Failing since 35 runs

* test_list_ids_parameter.py

 * ContextSuite context=TestListIdsParams>:setup Failing since 38 runs

* test_vm_life_cycle.py

 * test_10_attachAndDetach_iso Failing since 35 runs

* test_routers_network_ops.py

 * test_01_isolate_network_FW_PF_default_routes_egress_true Failing since 68 runs

 * test_02_isolate_network_FW_PF_default_routes_egress_false Failing since 68 runs

 * test_01_RVR_Network_FW_PF_SSH_default_routes_egress_true Failing since 67 runs

 * test_02_RVR_Network_FW_PF_SSH_default_routes_egress_false Failing since 67 runs


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_01_verify_libvirt
test_02_verify_libvirt_after_restart
test_03_verify_libvirt_attach_disk
test_04_verify_guest_lspci
test_05_change_vm_ostype_restart
test_06_verify_guest_lspci_again
test_static_role_account_acls
test_11_ss_nfs_version_on_ssvm
test_nested_virtualization_vmware
test_3d_gpu_support
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_vm_snapshots.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_loadbalance.py
test_routers.py
test_reset_vm_on_reboot.py
test_deploy_vms_with_varied_deploymentplanners.py
test_network.py
test_router_dns.py
test_login.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_metrics_api.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_volumes.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_disk_offerings.py

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by rhtyd** [comment]
LGTM.

**Comment by rhtyd** [comment]
One travis failure is not caused by this PR, but by the PR which removed secret key from user response. Merging now.

**Pull request merged**